.travis.yml

```diff
@@ -1,6 +1,8 @@
+before_install:
+  - gem update bundler
```

Comment:
I faced the problem on build failure at travis CI. (ref. https://travis-ci.org/mmozuras/pronto-scss/builds/101080452 https://s3.amazonaws.com/archive.travis-ci.org/jobs/101080453/log.txt)

This problem occurred because the bundler is old. (ref. http://stackoverflow.com/questions/34363675/refinery-nomethoderror-undefined-method-spec-for-nilnilclass )
So when before building, we have to update bundler :+1: